src/commercial.ts

```diff
@@ -18,13 +18,18 @@ void (async () => {
 		!commercialFeatures.adFree
 	) {
 		void import(
-			/* webpackChunkName: "consentless" */
-			'./init/consentless'
+			/* webpackChunkName: "consentless-advertising" */
+			'./init/consentless-advertising'
```

Comment:
This is very minor so feel free to ignore :) What do we gain anything by adding `-advertising` to all the chunks? Is it to help searching in dev tools?

Reply:
It was only to make the distinction between code that loads an ad and the ad free code really explicit to anyone looking at it for the first time. I'm not set on it at all - happy to go with the majority opinion!